tests/integrations/test_tornado.py

```diff
@@ -0,0 +1,108 @@
+import bugsnag
+import json
+from tests.utils import IntegrationTest
+from tornado.testing import AsyncHTTPTestCase
+import tests.fixtures.tornado.server
```

Comment:
`from tests.fixtures.tornado import server` should let you refer to just `server.make_app` on line 10